### PR TITLE
Export state code

### PR DESCRIPTION
**Pull request by aditim121**
Added validations in validateGenesis()

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,8 +8,10 @@ require (
 	github.com/cosmos/cosmos-sdk v0.34.4-0.20200530180557-ba70f4d4dc2e
 	github.com/gorilla/mux v1.7.4
 	github.com/pkg/errors v0.9.1
+	github.com/shopspring/decimal v1.2.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/viper v1.7.0
 	github.com/tendermint/tendermint v0.33.4
 	github.com/tendermint/tm-db v0.5.1
+	gopkg.in/validator.v2 v2.0.0-20200605151824-2b28d334fa05
```
Comment (deepanshutr) [inline go.mod:16]: The project already uses a validator https://github.com/asaskevich/govalidator, please remove this and use the old one

File: go.mod [hunk 2]
```diff
@@ -8,8 +8,10 @@ require (
 	github.com/cosmos/cosmos-sdk v0.34.4-0.20200530180557-ba70f4d4dc2e
 	github.com/gorilla/mux v1.7.4
 	github.com/pkg/errors v0.9.1
+	github.com/shopspring/decimal v1.2.0
```
Comment (deepanshutr) [inline go.mod:11]: unused import

File: modules/assets/genesis/genesis.go [hunk 3]
```diff
@@ -7,23 +7,39 @@ package genesis
...
 	"github.com/persistenceOne/persistenceSDK/schema/traits"
 	"github.com/persistenceOne/persistenceSDK/schema/types/base"
+	"gopkg.in/validator.v2"
 )
 
 type genesisState struct {
 	AssetList []mappables.InterNFT
 }
 
+//MapList := map[genesisState.AssetList]bool{}
```
Comment (deepanshutr) [inline modules/assets/genesis/genesis.go:22]: commit no commented code

File: modules/assets/genesis/genesis.go [hunk 4]
```diff
@@ -7,23 +7,39 @@ package genesis
...
 }
 
-func (genesisState genesisState) Validate() error { return nil }
+func (genesisState genesisState) Validate() error {
+	for _, asset := range genesisState.AssetList {
+		if errs := validator.Validate(asset); errs != nil {
+			return errs
+		}
+
+		if asset.GetID() == nil || asset.GetID().String() == ""{ return constants.EntityNotFound }
```
Comment (deepanshutr) [inline modules/assets/genesis/genesis.go:37]: checks already verified by validator

Comment (deepanshutr) [reply modules/assets/genesis/genesis.go:37]: correct everywhere

File: modules/classifications/genesis/genesis.go [hunk 5]
```diff
@@ -23,7 +25,25 @@ func (genesisState genesisState) Default() helpers.GenesisState {
...
-func (genesisState genesisState) Validate() error { return nil }
+
+func (genesisState genesisState) Validate() error {
+
+	for _, classification := range genesisState.ClassificationList {
+		if errs := validator.Validate(classification); errs != nil {
+			return errs
+		}
+
+		if classification.GetID() == nil { return constants.EntityNotFound}
```
Comment (deepanshutr) [inline modules/classifications/genesis/genesis.go:36]: validator checks for these conditions, unnecessar

File: go.mod (hunk 1 above)
Comment (aditim121) [reply go.mod]: Done. now using asaskevich/govalidator

File: go.mod (hunk 2 above)
Comment (aditim121) [reply go.mod]: Have removed this from go.mod

File: modules/assets/genesis/genesis.go (hunk 3 above)
Comment (aditim121) [reply modules/assets/genesis/genesis.go]: oops, have removed this

File: modules/assets/genesis/genesis.go (hunk 4 above)
Comment (aditim121) [reply modules/assets/genesis/genesis.go]: Done

File: modules/classifications/genesis/genesis.go (hunk 5 above)
Comment (aditim121) [reply modules/classifications/genesis/genesis.go]: Done.

**Pull request merged**